test/performance/search/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+seqan3_benchmark(search_benchmark.cpp)
+
```

Comment:
remove empty line

Reply:
> Looks good, just some minor comments!
> 
> Oh, and how long does the complete benchmark take to run?
81 seconds

Reply:
> I did no manage to get the library built in release with google benchmark.

What? Why? If it doesn't build, we cannot merge it.

Reply:
It does built, but the Library is built in Debug.
(***WARNING*** Library was built as DEBUG. Timings may be affected.)

Reply:
Well, if doesn't built in release mode, it's still broken :wink:

Reply:
The problem is I don't know how to configure it so the library gets built in release not that it does not built. 
cmake -DC_MAKE_BUILD_TYPE=Release ../../../seqan3/test/performance/
it complains:
  Manually-specified variables were not used by the project:
    C_MAKE_BUILD_TYPE

Reply:
> C_MAKE_BUILD_TYPE

Maybe try `CMAKE_BUILD_TYPE` instead  :wink:

Reply:
Ok I tried CMAKE_BUILD_TYPE. It seems to always cut of the first letter in the error message... weird.
cmake -CMAKE_BUILD_TYPE=Release ../../../seqan3/test/performance/
loading initial cache file MAKE_BUILD_TYPE=Release
CMake Error: Error processing file: MAKE_BUILD_TYPE=Release
....

Reply:
Ok now it works. I did not understand I was supposed to use -DCMAKE_BUILD_TYPE and still did not see my typo :sweat_smile:.